ingestion/src/metadata/ingestion/api/common.py

```diff
@@ -61,6 +61,7 @@ def allow_all(cls):
     def included(self, string: str) -> bool:
         try:
             for exclude in self.excludes:
+                print(f"{exclude} and {string}")
```

Comment:
Maybe just clear the print or change this for a logger

Reply:
Done.